frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -95,6 +102,28 @@ export const validateOperatingSystem: VmSettingsValidator = (field) => {
   return asValidationObject(`Select matching for: ${guestFullName}`, ValidationErrorType.Info);
 };
 
+const memoryValidation: VmSettingsValidator = (field, options) => {
+  const memValueGB = iGetFieldValue(field);
+  const validations = getTemplateValidations(options, getFieldId(VMSettingsField.MEMORY));
+  if (validations === [] || !memValueGB) {
+    return null;
+  }
+
+  // convert memory value to Bytes as it is in the validation rule
+  const memValueBytes = memValueGB * 1024 ** 3;
+
+  // filter all the 'min' validation rules
+  const minValidations = validations.filter((v) => 'min' in v);
```

Comment:
I like better running each check individually and looking for first success, then finding the min of all validations and then checking once. because it create a pattern of checking rules we can apply for all checks, e.g. on regex:
```
 validations.once( ... // do one min validation at a time )
```

```
 validations.once( ... // do one regex validation at a time )
```